src/components/Checkbox/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './index.css';
+
+
+function Checkbox({
+    children,
+    checked,
+    onChange
+}) {
+    return(
+            <label className='label'>
+                    <input className='checkbox' type='checkbox' checked={checked} onChange={onChange} />
+                {children}
```

Comment:
this is weird since html checkbox doesn't support children, in fact no `input` tag supports it, instead of having a prop called `children` and using it as the label text you can name it as `label` and pass it as the children of the `label` tag
```suggestion
                {label}
```